### DISCUSSION TITLE
clase backend

### DISCUSSION THREAD

**Discussion by edyy-1010** [Launch-X-Latam/MisionFrontEnd#541] [Team Posts]
alguien me puede decir si hubo clase de backend hoy no pude ingresar y no se como hacer

**Comment by Memotets** [comment]
No te preocupes, las clases se quedan grabadas en apartado de Teams, puedes ver la sesión cuando tengas la oportunidad...

**Comment by LeticiaOlivera** [comment]
Si hubo sesión el día de hoy, mañana es la próxima, pero quedan grabadas en Teams :)
